Add procedure howto restore elemental CRD resources (#477)
rancher/elemental: docs/restore.md

# Restore

> Follow this guide to restore from backup elemental nodes configuration together with Rancher.

Reviewer: ```suggestion
Follow this guide to restore an elemental node configuration from a backup with Rancher.
```

Author: fixed

## Prepare rancher-backup operator and backup files for restoring

> Go to official [Rancher documentation](https://docs.ranchermanager.rancher.io/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher) and make sure that rancher-bakup operator is installed and has access to backup files.

Reviewer: ```suggestion
Go to official [Rancher documentation](https://docs.ranchermanager.rancher.io/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/restore-rancher) and make sure that `rancher-bakup operator` is installed and has access to backup files.
```

Author: fixed

> ## Restore elemental with rancher-backup operator

Reviewer: ```suggestion
## Restore the elemental node configuration with rancher-backup operator
```

Author: fixed

> Restore backup from tarball by creating Restore object

Reviewer: ```suggestion
Create a `restore object` to restore the backup tarball:
```

Author: fixed
